tests/Controller/HomeControllerTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Tests\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class HomeControllerTest extends WebTestCase
+{
+    /**
+     * @dataProvider provider
+     * @param $path
+     */
+    public function testHome($path): void
```

Comment:
I guess we can type-hint here also :wink:

Reply:
fixed! thanks!